pydatastructs/graphs/algorithms.py

```diff
@@ -548,3 +550,61 @@ def strongly_connected_components(graph, algorithm):
         "isn't implemented for finding strongly connected components."
         %(algorithm, graph._impl))
     return getattr(algorithms, func)(graph)
+
+
+def random_walk(graph, start, path_length, operation, seed=44):
```

Comment:
How can I pass the parameters for operation?

Reply:
I will add `*args`, so that we can pass additional arguments if required.